schema.go

```diff
@@ -0,0 +1,43 @@
+package ipld
+
+import (
+	"bytes"
+	"io"
+	"os"
+
+	"github.com/ipld/go-ipld-prime/schema"
+	schemadmt "github.com/ipld/go-ipld-prime/schema/dmt"
+	schemadsl "github.com/ipld/go-ipld-prime/schema/dsl"
+)
+
+// LoadSchemaBytes is a shortcut for LoadSchema for the common case where
+// the schema is available as a buffer or a string, such as via go:embed.
+func LoadSchemaBytes(src []byte) (*schema.TypeSystem, error) {
+	return LoadSchema("", bytes.NewReader(src))
+}
+
+// LoadSchemaBytes is a shortcut for LoadSchema for the common case where
+// the schema is a file on disk.
+func LoadSchemaFile(path string) (*schema.TypeSystem, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return nil, err
+	}
+	defer f.Close()
+	return LoadSchema(path, f)
```

Comment:
do you want `path.BaseName` here?

Reply:
I figured not, e.g. if I run it on `foo/bar/baz.ipldsch`, the full path seems useful.

One possibility would be to do like most compilers, and make the path relative to the current directory for readability. But "current directory" feels like a weird concept for a library.